packages/web/src/hooks/useSyntaxHighlightingExtension.ts

```diff
@@ -30,28 +78,24 @@ export const useSyntaxHighlightingExtension = (language: string, view: EditorVie
 }
 
 export const getSyntaxHighlightingExtension = (language: string) => {
+    // maps linguist language defs to CodeMirror 6 language extensions
+    // and legacy CodeMirror 5 modes
+    // console.log(language.toLowerCase()); // uncomment to debug
```

Comment:
nit: we can probably just remove this commented out log, or change it to a `console.debug`

Reply:
removed.